client/components/phone-input/phone-number.ts

```diff
@@ -11,31 +14,33 @@ const LONGEST_NUMBER = '999999999999999';
 const LONGEST_NUMBER_MATCH = /9/g;
 export const MIN_LENGTH_TO_FORMAT = 3;
 
+const countries: Record< string, CountryData > = countriesRaw;
+const dialCodeMap: Record< string, string[] > = dialCodeMapRaw;
```

Comment:
Because it looks like the data file is auto-generated, I don't want to risk converting it to TypeScript, so I'm typing its data here instead.